CHANGELOG.md

```diff
@@ -1,446 +1,51 @@
-## [2.9.4](https://github.com/theKashey/react-focus-lock/compare/v2.9.3...v2.9.4) (2023-02-16)
-
-
-### Bug Fixes
-
-* update focus-lock to prevent error when accessing cross-origin frames. fixes [#241](https://github.com/theKashey/react-focus-lock/issues/241) ([6633d2b](https://github.com/theKashey/react-focus-lock/commit/6633d2bed7f61fe1b6ca9a573331568bbd0b63af))
-
-
-
-## [2.9.3](https://github.com/theKashey/react-focus-lock/compare/v2.9.2...v2.9.3) (2023-01-28)
-
-
-
-## [2.9.2](https://github.com/theKashey/react-focus-lock/compare/v2.9.1...v2.9.2) (2022-11-13)
-
-
-### Bug Fixes
-
-* suppress ref warning for host nodes, fixes [#123](https://github.com/theKashey/react-focus-lock/issues/123) ([fb14cb5](https://github.com/theKashey/react-focus-lock/commit/fb14cb58634b72bc16e968dc8fd3651992784b9d))
-
-
-
-## [2.9.1](https://github.com/theKashey/react-focus-lock/compare/v2.9.0...v2.9.1) (2022-05-07)
-
-
-
-# [2.9.0](https://github.com/theKashey/react-focus-lock/compare/v2.8.1...v2.9.0) (2022-05-01)
-
-
-### Bug Fixes
-
-* no more using SFC types ([ca9e5e8](https://github.com/theKashey/react-focus-lock/commit/ca9e5e8ae1bbb4ee252a11490f95c2dcc2f76efd))
-* update dependnecies. fixes [#210](https://github.com/theKashey/react-focus-lock/issues/210) and fixes [#212](https://github.com/theKashey/react-focus-lock/issues/212) ([a3895ce](https://github.com/theKashey/react-focus-lock/commit/a3895cec02b2a748f85424c2dabaf9c8b698b76c))
-
-
-
-## [2.8.1](https://github.com/theKashey/react-focus-lock/compare/v2.8.0...v2.8.1) (2022-02-14)
+## 2.9.5 (2023-10-11)
```

Comment:
can you please stash changes in this file 🙏

Reply:
Sure, done!